### ISSUE TITLE
All Static Props are being converted to World Geometry regardless of "Bake To World" property

### ISSUE THREAD

**Issue by gvarados1** [Facepunch/sbox-issues#1270]
All static props seem to be converted to world geo on compile, regardless of if "Bake To World" is set.

**Comment by robotboy655** [comment]
I cannot seem to reproduce this. How are you telling the difference?

**Comment by DoctorGurke** [comment]
The models are baked into the world nodes. It's easiest to test when using custom content, since prop statics won't need their models included. (this does not apply to materials of course)

**Comment by robotboy655** [comment]
![image](https://user-images.githubusercontent.com/3299036/142656248-502e57b4-d92d-407b-838f-fac0be57b8c5.png)

![image](https://user-images.githubusercontent.com/3299036/142656382-961f0a34-c228-4fe6-9fb8-7d7d5bec2d55.png)

Fast compile, rust models.

Models will be force baked if they have skeleton data.

**Comment by DoctorGurke** [comment]
Yep seems like they don't actually bake after all. Is this a difference to the hla compilers? could've sworn hla baked any static prop.
I suppose it'd be reasonable to ask for that option to be true by default, though
